### PR TITLE
Completed Intermediary Database Module.

### PR DESCRIPTION
**Pull request by Bhavesh00**

### CODE REVIEW

File: intermediary_database.py [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+
+def mysql_disconnect():
+    """Closes the MySQL database connection.
+    """
+    connection.close()
+
+def close_ssh_tunnel():
+    """Closes the SSH tunnel connection.
+    """
+    tunnel.close
```
Comment (ashuk203) [inline intermediary_database.py]: Did you mean `tunnel.close()` here?

File: crawl_OAG.py [hunk 2]
```diff
@@ -97,49 +107,188 @@ def crawl_helper(professor, university, file):
...
+
+def mysql_disconnect():
+    """Closes the MySQL database connection.
+    """
+    connection.close()
+
+def close_ssh_tunnel():
+    """Closes the SSH tunnel connection.
+    """
+    tunnel.close
```
Comment (ashuk203) [inline crawl_OAG.py]: Same thing here, did you mean `tunnel.close()`?

File: crawl_OAG.py [hunk 3]
```diff
@@ -97,49 +107,188 @@ def crawl_helper(professor, university, file):
...
+            temp_dict["pubs"] = author_json["pubs"]
+
+        else:
+            temp_dict["pubs"] = ""
+            
+        authors = authors.append(temp_dict, ignore_index=True)
+    
+    return authors
+
+def open_ssh_tunnel(verbose=False):
```
Comment (ashuk203) [inline crawl_OAG.py]: Consider modularizing and extracting the functions `open_ssh_tunnel`, `mysql_connect`, `run_query`, `mysql_disconnect`, and `close_ssh_tunnel` into a separate file as it looks like you reuse them in the `intermediary_database.py` file. 

This way, you don't have to copy past the functions and can simply import them from one file.

**Review by ashuk203 — CHANGES_REQUESTED** [review]
Looks good! Just need you to address the few comments below.

File: intermediary_database.py (hunk 1 above)
Comment (Bhavesh00) [reply intermediary_database.py:142]:
```suggestion
    tunnel.close()
```

File: crawl_OAG.py (hunk 2 above)
Comment (Bhavesh00) [reply crawl_OAG.py:284]:
```suggestion
    tunnel.close()
```

File: crawl_OAG.py (hunk 3 above)
Comment (Bhavesh00) [reply crawl_OAG.py]: Added sql_helper.py file to solve this issue.

File: crawl_OAG.py (hunk 2 above)
Comment (Bhavesh00) [reply crawl_OAG.py]: Updated to be tunnel.close()

File: intermediary_database.py (hunk 1 above)
Comment (Bhavesh00) [reply intermediary_database.py]: Updated to be tunnel.close()

**Pull request merged**